### ISSUE TITLE
Paperback: JSError Error [PaperbackError(300)] JSError Error: JSON PARSE ERROR! You've like set too many filters in this source's settings, remove some to see results! - parseJson@ @:712:28

### ISSUE THREAD

**Issue by kratoscortez** [TheNetsky/community-extensions#36]
Hentai2Read, MangaDex, MangaPlus, nhentai are all getting this error. I have removed and readded the source and issue still occurs
![IMG_4231](https://github.com/TheNetsky/community-extensions/assets/161261402/53d31ab6-313b-45f4-9f0a-9eb3d99f7eef)

**Comment by Ivanmatthew** [comment]
Which filters did you apply?

**Comment by kratoscortez** [comment]
I haven’t applied any, I have also tried removing the existing ones. I went to settings, extentions, selected the extension, clicked content settings and there was “additional” arguments in there so I removed them but even if I reset to default I get the same error

**Comment by Ivanmatthew** [comment]
Can you send the logs?

**Comment by kratoscortez** [comment]
[04-03.Paperback.txt](https://github.com/TheNetsky/community-extensions/files/14485716/04-03.Paperback.txt)

Here are the app logs please let me know if you need any other info

**Comment by Ivanmatthew** [comment]
Looking at the source code this only affects nhentai.
I think you’ve been ratelimited of some sorts, as the parsing part indeed fails and nothing else.

Might just be an issue of waiting. If issue still persists do tell.

**Comment by SlayerOfDemons1** [comment]
This is happening to me as well for the past few days.

However i found that adding extra arguments gets me new results other than the ones already loaded before, i hope that’s of help to find a solution to the issue.

**Comment by kratoscortez** [comment]
I have tried adding 3 more arguments and still happens, I will give it a week and report back

**Comment by kratoscortez** [comment]
Issue is still happening on and off, is there any other troubleshooting I can do here to resolve it.

**Comment by kratoscortez** [comment]
> Looking at the source code this only affects nhentai.
> 
> I think you’ve been ratelimited of some sorts, as the parsing part indeed fails and nothing else.
> 
> 
> 
> Might just be an issue of waiting. If issue still persists do tell.

Hey @Ivanmatthew issue is still happening on and off is there any troubleshooting I can do?

**Comment by Ivanmatthew** [comment]
> > Looking at the source code this only affects nhentai.
> > I think you’ve been ratelimited of some sorts, as the parsing part indeed fails and nothing else.
> > Might just be an issue of waiting. If issue still persists do tell.
> 
> Hey @Ivanmatthew issue is still happening on and off is there any troubleshooting I can do?

I will look into it and relay back to you.

**Comment by Ivanmatthew** [comment]
> > > Looking at the source code this only affects nhentai.
> > > I think you’ve been ratelimited of some sorts, as the parsing part indeed fails and nothing else.
> > > Might just be an issue of waiting. If issue still persists do tell.
> > 
> > 
> > Hey @Ivanmatthew issue is still happening on and off is there any troubleshooting I can do?
> 
> I will look into it and relay back to you.

@kratoscortez I can not reproduce it under any circumstance, if you have time, report your issue on Netsky's discord and maybe we can go in depth with MitM analysis

**Comment by kratoscortez** [comment]
I went ahead and posted in the support channel, thanks so much I am happy to do any troubleshooting suggested to me as I just would like to get it working

**Comment by 2Lazy2Move** [comment]
Yeah @Ivanmatthew even in the server #website-statuses it says "Error: Access Denied! Unable to check!"

**Comment by Sally-108** [comment]
i faced this problem in magakakalot source too
